src/expressServer.js

```diff
@@ -154,10 +177,7 @@ io.on('connection', (socket) => {
   });
 });
 
-// Serving the documentation page
-app.get('/tests', (req, res) => {
-  const documentationFilePath = path.join(__dirname, 'public', 'tests.html');
-  res.sendFile(documentationFilePath);
-});
```

Comment:
Are you intentionally removing this route for the documentation page?

Reply:
returned, it by mistake